src/servers/src/influxdb.rs

```diff
@@ -132,7 +162,11 @@ monitor2,host=host4 cpu=66.3,memory=1029 1663840496400340003";
             lines: lines.to_string(),
         };
 
-        let requests: RowInsertRequests = influxdb_req.try_into().unwrap();
+        let catalog_manager = MemoryCatalogManager::new() as _;
```

Comment:
It's better to add a test to cover `auto_align_precision=true`. You can use a mock catalog manager like `MemoryCatalogManager`.